File: pinyougou-manager/pinyougou-manager-interface/src/main/java/com/pinyougou/manager/service/BrandService.java
package com.pinyougou.manager.service;

import com.pinyougou.common.PageResult;
import com.pinyougou.pojo.TbBrand;

import java.util.List;

public interface BrandService {
    List<TbBrand> findAll();

    PageResult findPage(int pageNum,int pageSize);

    void add(TbBrand tbBrand);

    TbBrand findOne(Long id);

    void update(TbBrand tbBrand);

    void dele(Long[] ids);

    PageResult search(Integer pageNum, Integer pageSize, TbBrand tbBrand);
}

File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService', function ($http) {
    this.findAll = function () {
        return $http.get('../brand/findAll');
    }

    this.findPage = function (pageNum,pageSize) {
        return $http.get('../brand/findPage?pageNum=' + pageNum + '&pageSize=' + pageSize);
    }

    this.update=function (entity) {
        return $http.post('../brand/update', entity);
    }

    this.add=function (entity) {
        return $http.post('../brand/add', entity);
    }

    this.findOne=function (id) {
        return $http.get('../brand/findOne?id=' + id);
    }

    this.dele=function (selectId) {
        return $http.get('../brand/dele?ids=' + selectId);
    }

    this.search=function (pageNum,pageSize,searchEntity) {
        return $http.post("../brand/search?pageNum=" + pageNum + "&pageSize=" + pageSize, searchEntity);
    }

})